java/openmldb-taskmanager/src/test/scala/com/_4paradigm/openmldb/taskmanager/TestSparkLauncherUtil.scala

```diff
@@ -37,8 +37,7 @@ class TestSparkLauncherUtil extends FunSuite {
     Thread.sleep(3000)
 
     // Submit successfully and get final state
-    // TODO: get lost state since of lack of log4j jar, more info in sparkAppHandle.getError
-    assert(sparkAppHandle.getState.isFinal)
+    assert(sparkAppHandle.getState == SparkAppHandle.State.FINISHED)
```

Comment:
It is expected to be State.LOST now.
We can assert sparkAppHandle.getError is empty.

Reply:
Resolved~